app/Console/Commands/RecalculateStatusDistance.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Http\Controllers\Backend\Transport\TrainCheckinController;
+use App\Models\Status;
+use Illuminate\Console\Command;
+
+class RecalculateStatusDistance extends Command
+{
+    protected $signature   = 'trwl:recalculateStatusDistance {id*} {--polyline}';
```

Comment:
Wofür ist der `--polyline`, wenn in L21 reset Polyline immer `true` ist?

Reply:
Künftige Erweiterbarkeit oder so. 

Wollte gestern Abend noch einbauen, dass in dem Fall ne neue Polyline erstellt wird, statt sie zu löschen, aber ich war zu kaputt. Hätte das dann die Tage™ nachgereicht.

Reply:
Hab das Argument mal entfernt.